<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>584</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>800</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Orquesta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>1</number>
       </property>
       <widget class="QGroupBox" name="groupbox_terminals">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Terminals</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="OrListWidget" name="listwidget"/>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupbox_cwd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Logs path</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="OrLogsListView" name="logslistview"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="OrSciScintilla" name="sciscintilla">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>18</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_exit"/>
    <addaction name="action_new"/>
   </widget>
   <widget class="QMenu" name="menu_operation">
    <property name="title">
     <string>&amp;Operations</string>
    </property>
    <addaction name="action_open_terminals"/>
    <addaction name="action_send_move_next"/>
    <addaction name="action_send"/>
    <addaction name="action_close_terminals"/>
    <addaction name="separator"/>
    <addaction name="action_edit_var_file"/>
    <addaction name="action_edit_config"/>
    <addaction name="action_open_local_dir"/>
    <addaction name="action_open_local_term"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_selection">
    <property name="title">
     <string>&amp;Selection</string>
    </property>
    <addaction name="action_select_all"/>
    <addaction name="action_unselect_all"/>
    <addaction name="action_invert"/>
   </widget>
   <widget class="QMenu" name="menu_templates">
    <property name="title">
     <string>&amp;Templates</string>
    </property>
    <addaction name="action_save_as_template"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_project"/>
   <addaction name="menu_selection"/>
   <addaction name="menu_operation"/>
   <addaction name="menu_templates"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_exit"/>
   <addaction name="separator"/>
   <addaction name="action_new"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_open_terminals"/>
   <addaction name="separator"/>
   <addaction name="action_edit_var_file"/>
   <addaction name="action_edit_config"/>
   <addaction name="action_open_local_term"/>
   <addaction name="action_open_local_dir"/>
  </widget>
  <action name="action_save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/img/save.png</normaloff>:/toolbar/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/img/exit.png</normaloff>:/toolbar/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_select_all">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Select all terminals</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="action_unselect_all">
   <property name="text">
    <string>&amp;Unselect all terminals</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_open_terminals">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/img/remote.png</normaloff>:/toolbar/img/remote.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open selected terminals</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_send_move_next">
   <property name="text">
    <string>Execute current line and move next</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_send">
   <property name="text">
    <string>Execute current line</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/img/new.png</normaloff>:/toolbar/img/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_invert">
   <property name="text">
    <string>&amp;Invert selection</string>
   </property>
   <property name="toolTip">
    <string>Invert selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="action_close_terminals">
   <property name="text">
    <string>Close selected terminals</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="action_save_as_template">
   <property name="text">
    <string>&amp;Save as template</string>
   </property>
  </action>
  <action name="action_edit_var_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/img/var.png</normaloff>:/toolbar/img/var.png</iconset>
   </property>
   <property name="text">
    <string>Edit var file</string>
   </property>
  </action>
  <action name="action_edit_config">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/img/config.png</normaloff>:/toolbar/img/config.png</iconset>
   </property>
   <property name="text">
    <string>Edit config file</string>
   </property>
   <property name="toolTip">
    <string>Edit config file</string>
   </property>
  </action>
  <action name="action_open_local_dir">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/img/directory.png</normaloff>:/toolbar/img/directory.png</iconset>
   </property>
   <property name="text">
    <string>Open file explorer in logs path</string>
   </property>
   <property name="toolTip">
    <string>Open file explorer in logs path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="action_open_local_term">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/img/terminal.png</normaloff>:/toolbar/img/terminal.png</iconset>
   </property>
   <property name="text">
    <string>Open local terminal in current log path</string>
   </property>
   <property name="toolTip">
    <string>Open local terminal in current log path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="action_restart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/img/reset.png</normaloff>:/toolbar/img/reset.png</iconset>
   </property>
   <property name="text">
    <string>_restart</string>
   </property>
   <property name="toolTip">
    <string>Restart</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
  <customwidget>
   <class>OrSciScintilla</class>
   <extends>QsciScintilla</extends>
   <header>widgets.orsciscintilla</header>
  </customwidget>
  <customwidget>
   <class>OrListWidget</class>
   <extends>QListWidget</extends>
   <header>widgets.orlistwidget</header>
  </customwidget>
  <customwidget>
   <class>OrLogsListView</class>
   <extends>QListView</extends>
   <header>widgets.orlogslistview</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listwidget</tabstop>
  <tabstop>sciscintilla</tabstop>
  <tabstop>logslistview</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
